deploy/template.yaml

```diff
@@ -25,7 +25,19 @@ Parameters:
     Type: String
     Default: "https://api.os.uk/search/places/v1/postcode"
 
+  OrdnanceSurveyAPIKey:
+    Description: Ordnance Survey API Key
+    Type: String
+    Default: testapikey
+
+
 Conditions:
+  UseCodeSigning:
```

Comment:
This condition is evaluated in the globals block below, so you shouldn't have to apply it to your function